bundles/bundles.go

```diff
@@ -233,10 +248,16 @@ func (e OCIImageRunner) Install(config *BundleConfig) error {
 	if err != nil {
 		return err
```

Comment:
found out that GetImage is already handling empty platform case. So we dont need to handle it here. https://github.com/nianyush/kairos-sdk/blob/7d11ed67c526ce2477bf2103a92fb405a90ff36a/utils/image.go#L80